index.js

```diff
@@ -61,7 +104,8 @@ module.exports = function inspect_ (obj, opts, depth, seen) {
         for (var i = 0; i < obj.length; i++) {
             xs[i] = has(obj, i) ? inspect(obj[i], obj) : '';
         }
-        return '[ ' + xs.join(', ') + ' ]';
+        return listFmt('Array', false, xs,
```

Comment:
why is the "size" param sometimes `false` and sometimes `null`?

I'd kind of expect arrays to pass their length here, even if the "objType" chose to omit it (I do see your comment in the function above)